libbpf-tools/statsnoop.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __STATSNOOP_H
+#define __STATSNOOP_H
+
+#define TASK_COMM_LEN   16
+#define NAME_MAX        255
+
+struct value_t {
+	const char *pathname;
+};
+
+struct event_t {
```

Comment:
I think the `_t` suffix is a reserved suffix for POSIX APIs. I think these can just be called `struct value` and `struct event`

Reply:
OK, will do.

I find some of the tools already use `_t` suffix.